recipes/upstream/digitalocean/rke/main.tf

```diff
@@ -23,6 +23,7 @@ module "rke" {
   kube_config_path     = pathexpand(var.kube_config_path)
   kube_config_filename = var.kube_config_filename
   kubernetes_version   = var.kubernetes_version
+  additional_hostnames = [module.upstream-cluster.k8s_api_loadbalancer_ip]
```

Comment:
Plz make this conditional

Reply:
Handled this in the upstream module so that an empty list will be passed to the `sans` field when module.upstream-cluster.k8s_api_loadbalancer_ip returns a null value.

https://github.com/rancher/tf-rancher-up/pull/147/commits/25311fc91362bb86a1abddd6a9f44be29d0a6786